packages/react/src/side-nav/SideNav.tsx

```diff
@@ -18,21 +18,24 @@ export type SideNavProps = {
 	activePath: string;
 	/** Used as the title of the expand/collapse trigger on smaller screen sizes. */
 	collapseTitle: string;
-	/** If SideNav is placed on 'bodyAlt' background, please set this to 'bodyAlt'. */
-	background?: CollapsingSideBarBackground;
 	/** The list of links. */
 	items: SideNavMenuItemType[];
 	/** The title is placed at the top of the list of links. */
 	title: string;
+	/** If SideNav is placed on 'bodyAlt' background, please set this to 'bodyAlt'. */
+	background?: CollapsingSideBarBackground;
+	/** The variant to use for nested navigation items. */
+	nestedItemsVariant?: 'always-open' | 'open-on-nav';
```

Comment:
I would use camelCase like we have elsewhere such as StatusBadge

Reply:
- [ ] camelCase options